src/ert/config/gen_data_config.py

```diff
@@ -122,10 +122,10 @@ def from_config_dict(cls, config_dict: ConfigDict) -> Optional[Self]:
     def read_from_file(self, run_path: str, iens: int, iter: int) -> polars.DataFrame:
         def _read_file(filename: Path, report_step: int) -> polars.DataFrame:
             try:
-                data = np.loadtxt(_run_path / filename, ndmin=1)
+                data = np.loadtxt(filename, ndmin=1)
```

Comment:
:heart: I also stumbled on this while working on something unrelated, and forgot to follow up with more investigation on why it was so strange. I am glad to see it fixed.

Reply:
Yes, it works because adding two abs path together ignores the shared parents, I did not know that, and once I started using `_run_path` as a relative path in the tests, it stopped working